.codecov.yml

```diff
@@ -24,6 +24,11 @@ coverage:
 ignore:
   - cmd
 
+# https://docs.codecov.com/docs/flags#recommended-automatic-flag-management
+flag_management:
+  default_rules:
+    carryforward: false
+
```

Comment:
Can we close https://github.com/FerretDB/FerretDB/issues/2106 then?

Reply:
I don't know yet. I briefly checked codecov.io documentation, and could no longer find the feature I was talking about… I will take another look after we have a few builds with this new config.